includes/discovery/bgp-peers.inc.php

```diff
@@ -11,15 +11,94 @@
     if (file_exists(Config::get('install_dir') . "/includes/discovery/bgp-peers/{$device['os']}.inc.php")) {
         include Config::get('install_dir') . "/includes/discovery/bgp-peers/{$device['os']}.inc.php";
     }
+    if ($device['os'] == 'firebrick') {
```

Comment:
Could you please put this block in its own file (firebrick.inc.php) ?
You should also define ```  $bgpLocalAs = $bgpPeers[0][array_keys($bgpPeers[0])[0]]['fbBgpPeerRemoteAS']; ```in the file, to avoid the condition you created around line 91.

Reply:
I've moved this to its own file.